composer.json

```diff
@@ -4,15 +4,16 @@
     "license": "MIT",
     "require": {
         "php": ">= 5.6",
-        "nikic/php-parser": "^2.0-beta",
+        "nikic/php-parser": "^2.0",
         "phpdocumentor/reflection-docblock": "^2.0",
-        "phpdocumentor/type-resolver": "^0.1.5",
+        "phpdocumentor/type-resolver": "^0.1.6",
         "zendframework/zend-code": "^3.0",
         "jeremeamia/superclosure": "^2.2"
     },
     "require-dev": {
         "fabpot/php-cs-fixer": "^1.8",
-        "phpunit/phpunit": "^4.6"
+        "phpunit/phpunit": "^4.6",
+        "phly/changelog-generator": "^2.1"
```

Comment:
This isn't needed for normal dev environments. You should install it separately on your machine